contrib/clojure-package/examples/bert/project.clj

```diff
@@ -16,13 +16,15 @@
 ;;
 
 
-(defproject bert-qa "0.1.0-SNAPSHOT"
-  :description "BERT QA Example"
-  :plugins [[lein-cljfmt "0.5.7"]]
+(defproject bert "0.1.0-SNAPSHOT"
+  :description "BERT Examples"
+  :plugins [[lein-cljfmt "0.5.7"]
+            [lein-jupyter "0.1.16" :exclusions [org.clojure/tools.nrepl org.clojure/clojure org.codehaus.plexus/plexus-utils org.clojure/tools.reader]]]
```

Comment:
Just FYI I had to comment out the `lein-jupyter` plugin in order to have a working REPL with CIDER version 0.19.0 Raleigh.

Reply:
I ran into the some issues as well with cider and the older version of lein-jupyter playing well together. I added a note in the README and in the project.clj